### PR TITLE
Remoting component for server logging

### PR DESCRIPTION
**Pull request by hendrikebbers**
- adding first remoting component for logging
- show server logging on client

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/804)

### CODE REVIEW

File: platform-extras/dolphin-platform-logger/src/main/java/com/canoo/platform/logging/DolphinLoggerConfiguration.java [hunk 1]
```diff
@@ -38,6 +42,19 @@
 
     private HttpURLConnectionFactory connectionFactory = new DefaultHttpURLConnectionFactory();
 
+    public void setLevel(String name, Level level) {
```
Comment (giftkugel) [inline platform-extras/dolphin-platform-logger/src/main/java/com/canoo/platform/logging/DolphinLoggerConfiguration.java]: Add `final` keyword

File: platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/bridges/SimpleDolphinLogger.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
     private final Level level;
 
     public SimpleDolphinLogger(final DolphinLoggerConfiguration configuration) {
-        this.level = Objects.requireNonNull(configuration.getGlobalLevel());
+        this.level = Level.TRACE;
```
Comment (giftkugel) [inline platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/bridges/SimpleDolphinLogger.java]: Please comment this. Is `TRACE` the correct log level?

File: platform-extras/dolphin-platform-logger/src/main/java/com/canoo/platform/logging/DolphinLoggerConfiguration.java (hunk 1 above)
Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger/src/main/java/com/canoo/platform/logging/DolphinLoggerConfiguration.java]: done

File: platform-extras/dolphin-platform-logger-widget-client-javafx/src/main/java/com/canoo/platform/logger/client/view/LogListViewController.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import javafx.scene.control.ListView;
+
+import static com.canoo.platform.logger.LoggerRemotingConstants.LOG_LIST_CONTROLLER_NAME;
+import static com.canoo.platform.logger.LoggerRemotingConstants.UPDATE_ACTION;
+
+public class LogListViewController extends AbstractViewController<LogListBean> {
+
+    private final ListView<LogMessage> listView;
+
+    public LogListViewController(ClientContext clientContext) {
```
[the remark this answers is not in the document]
Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger-widget-client-javafx/src/main/java/com/canoo/platform/logger/client/view/LogListViewController.java]: done

File: platform-extras/dolphin-platform-logger-widget-client-javafx/src/main/java/com/canoo/platform/logger/client/view/LogListViewController.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                try {
+                    Thread.sleep(2_000);
+                } catch (InterruptedException e) { e.printStackTrace();}
+                clientConfiguration.getUiExecutor().execute(() -> invoke(UPDATE_ACTION));
+            }
+        });
+    }
+
+    private LogMessage convertBean(final LogEntryBean bean) {
+        final LogMessage message = new LogMessage();
```
[the remark this answers is not in the document]
Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger-widget-client-javafx/src/main/java/com/canoo/platform/logger/client/view/LogListViewController.java]: done

File: platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/DolphinLoggerFactory.java [hunk 5]
```diff
@@ -82,10 +77,7 @@ public synchronized void configure(final DolphinLoggerConfiguration configuratio
 
         for(DolphinLogger logger : loggerMap.values()) {
```
[the remark this answers is not in the document]
Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/DolphinLoggerFactory.java]: done

File: platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/bridges/SimpleDolphinLogger.java (hunk 2 above)
Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/bridges/SimpleDolphinLogger.java]: done

Comment (hendrikebbers) [reply platform-extras/dolphin-platform-logger/src/main/java/com/canoo/impl/dp/logging/bridges/SimpleDolphinLogger.java]: Currently no level is needed in this class. Maybe we add later a special min level for console logging.

**Pull request merged**